### DISCUSSION TITLE
Apple Silicon / macOS Big Sur Support

### DISCUSSION THREAD

**Discussion by oliexe** [castlabs/electron-releases#76] [Q&A] [answered]
Hi Guys,

As the new MACs are ARM-based I am opening the discussion about support for Apple Silicon.
Electron currently has darwin-arm64 build that is running under Apple Silicon - https://www.electronjs.org/blog/apple-silicon

Do we have any news regarding running the Widevine under ARM Mac ? Is that something You are considering in the future?
Thanks !

**Comment by khwaaj** [comment]
Hi Ondrej,

We are aware of the new Apple hardware and this is is something we are planning to support. I know the Widevine team is working on a `darwin-arm64` CDM, so once that is available we will look into what it takes to get ECS and EVS working on the new platform.

**Comment by oliexe** [comment]
Thanks Emil !,

Just a note: Can at least confirm that our app is functioning on Big Sur (x86).

**Comment by khwaaj** [comment]
I can also confirm that the x64 build works through Rosetta on M1 hardware, including Widevine protected media playback. I've not had the chance to do extensive testing so there may still be areas that have issues, but at least it is a good sign, and it should allow for a stop-gap solution until we can have a native arm64 build.

**Comment by stewartlord** [comment]
Any updates on this? Anything I can do to help? Looking forward to darwin-arm64 support 🎉

**Reply by khwaaj** [reply]
Unfortunately there is not much to do but wait. ECS has a port ready and once an official darwin-arm64 CDM is published for 3rd-party browsers we will start publishing these releases as well. The new CDM is currently in testing and provided there are no unexpected setbacks an official release should not be far off.

**Reply by stewartlord** [reply]
Thanks for the update! Sounds like we won't have to wait too long 👍

**Reply by jeffcunat** [reply]
It seems that the CDM is available for download now :  
https://r3---sn-4g5ednld.gvt1.com/edgedl/widevine-cdm/4.10.2209.0-mac-arm64.zip

Is it the right one for Apple Silicon ?

**Reply by khwaaj** [reply]
Yes, the new CDM, including support for `mac-arm64`, was pushed to official channels yesterday and became globally available late last night (CET). We have ECS builds in progress and will start publishing releases that include binaries for Apple silicon shortly.

**Comment by khwaaj** [comment, marked as answer]
With the new CDMs officially available from Widevine we today published the first batch of [releases](https://github.com/castlabs/electron-releases/releases) supporting Apple Silicon for the relevant release series, as follows:
* `v11.3.0-wvvmp` and later
* `v12.0.0-wvvmp` and later
* `v13.0.0-wvvmp-beta.2` and later

Please take them for a spin and let us know of any issues you encounter.

**Reply by pnegahdar** [reply]
That was fast! Just ran and built 11.4.0 and everything ran beautifully! Thank you all for the hard work!
